### PR TITLE
fix(sass): add support for sass syntax and update dependencies

### LINKED ISSUE #17: Sass/Scss support
The `sass` and `scss` syntax are not supported.

**NOTE**: The `scss` syntax is most likely supported only if it's 1:1 with plain CSS.

### PR DESCRIPTION
**Pull request by ohkimur**
This PR fixes #17 issue.

### CODE REVIEW

**Comment by ohkimur** [comment]
@Poimen I fixed the sass syntax issue. If you want you can write some tests for it. I also noticed that when writing utility classes using `@apply` the HMR doesn't work properly (or something similar). The component is remounted but, the styles that are applied in the css/sass/scss file are not applied. What's interesting is that on page reload the code works as intended. So this should be further investigated.

File: src/plugin.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export function buildEnd(err?: Error): void {
 }
 
 function useStyleSheetTransform(filename: string) {
-  return filename.match(/\.s?css/);
+  return filename.match(/\.s?(a|c)ss/);
```
Comment (Poimen) [inline src/plugin.ts:21]: Nice - can't believe I forgot about that 🙈

**Comment by Poimen** [comment]
Thanks for the contrib 👍 

On the HMR issue, this is a bug bear - did the `tailwindHMR()` help at all? (https://github.com/Poimen/stencil-tailwind-plugin#hmr-considerations)

**Comment by ohkimur** [comment]
> 

If I properly understand, at the moment `tailwindHMR()` only watches for the `.tsx` files. But, really it should also watch the CSS files. When using the `@apply` directive in the CSS files it should detect the changes and update the right module accodingly.

**Comment by ohkimur** [comment]
@Poimen I see the `main-build` is failing. Is there something I can do about that?

**Comment by ohkimur** [comment]
@Poimen Can we merge this into the main branch?

**Comment by Poimen** [comment]
Hi - sorry, I didn't get a notification message about your reply's 🙈 only got one now with your latest reply 😥 apologies for that.

So the build is failing on the `npm ci` command - so I'm guessing something to do with updates to packages.

If you revert those, we can merge...

**Comment by Poimen** [comment]
Released in `1.3.0`

**Pull request merged**